specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -11549,6 +11553,210 @@
+            "Failed",
+            "Completed"
+          ],
+          "x-ms-enum": {
+            "name": "RebootStatus",
+            "modelAsString": true
+          }
+        },
+        "notSelectedPatchCount": {
+          "type": "integer",
+          "readOnly": true,
+          "format": "int32",
+          "description": "The number of all available patches but not going to be installed because it didn't match a classification or inclusion list entry."
+        },
+        "excludedPatchCount": {
+          "type": "integer",
+          "readOnly": true,
+          "format": "int32",
+          "description": "The number of all available patches but excluded explicitly by a customer-specified exclusion list match."
+        },
+        "pendingPatchCount": {
+          "type": "integer",
+          "readOnly": true,
+          "format": "int32",
+          "description": "The number of all available patches expected to be installed over the course of the patch installation operation."
+        },
+        "installedPatchCount": {
+          "type": "integer",
+          "readOnly": true,
+          "format": "int32",
+          "description": "The number of all successfully installed patches."
+        },
+        "failedPatchCount": {
+          "type": "integer",
+          "readOnly": true,
+          "format": "int32",
+          "description": "The number of patches failed to installed."
+        },
+        "startTime": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The UTC timestamp when the operation began."
+        },
+        "lastModifiedTime": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The UTC timestamp when the operation began."
+        },
+        "startedBy": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The person or system account that started the operation"
+        },
+        "error": {
+          "$ref": "#/definitions/ApiError",
+          "readOnly": true,
+          "description": "The errors that were encountered during execution of the operation. The details array contains the list of them."
+        }
+      },
+      "description": "Describes the properties of an virtual machine instance view for last installed patch summary."
+    },
+    "ConfigurationStatusSummary": {
+      "properties": {
+        "patchModeProvisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Describes the enablement status of the specified patchMode <see cref=\"InGuestPatchMode\"/> on a VM.. \"Unknown\" is the state when the status is unknown. \"Pending\" is the state that tells Changes made to the patch configuration in the VM model have not yet been completed on the VM. \"Ready\" state shows that the VM is in the patch configuration state specified by its VM model.",
```

Comment:
"Describes the enablement status of the specified patchMode of a VM.. \"Unknown\" is the state when the status is unknown. \"Pending\" indicates that required configuration of the VM has not yet completed. \"Ready\" indicates that the VM is configured as specified by its patchMode."

Side note: Don't copy in crefs from the code comments